examples/README.txt

```diff
@@ -1,7 +1,7 @@
 .. _auto_examples:
 
-Examples
-========
+User guide
+==========
 
 Below is a gallery of narrated notebook examples on how and why to use skore.
-They also serve as our user guide.
+They serve as our user guide.
```

Comment:
I have a really strong opinion on this: by not forcing ourself to write the user guide, we are just going to drop a couple of line of example and state that this is good enough.

We should really write a user guide to give the perspective on our tools.

Reply:
Yes, we are thinking of doing a user guide in addition to an examples tab, but do we currently have enough features for a user guide?

Reply:
Created a separate issue: https://github.com/probabl-ai/skore/issues/1017